ui/app/helpers/supported-secret-backends.js

```diff
@@ -7,18 +7,19 @@ import { helper as buildHelper } from '@ember/component/helper';
 
 const SUPPORTED_SECRET_BACKENDS = [
```

Comment:
I'm a sucker for alphabetizing lists like this one. That's why you see lots of changes. The only true change is the addition of azure.